python/ray/air/checkpoint.py

```diff
@@ -246,6 +246,25 @@ def _metadata(self, metadata: _CheckpointMetadata):
         for attr, value in metadata.checkpoint_state.items():
             setattr(self, attr, value)
 
+    @property
+    def path(self) -> Optional[str]:
+        """Return path to persistent storage location of the checkpoint.
```

Comment:
I think the pydoc for this needs to be unified with URI below, since they are returning basically the same metadata just in slightly different output formats.